### DISCUSSION TITLE
Anyway to make OpenID Login the default page without the email login?

### DISCUSSION THREAD

**Discussion by mikekuzak** [danny-avila/LibreChat#1273] [Troubleshooting] [answered]
After configuring OpenID there should be a way to set the default login page to reroute to /oauth/openid which would be like SSO. 
I tried to use a Traefik reverse proxy and rewrite the path but getting too many redirect errors.

**Comment by danny-avila** [comment]
Would it suffice to have the default login page but omitting the email login? Kind of like this? 
![image](https://github.com/danny-avila/LibreChat/assets/110412045/793329ea-69aa-4468-9906-0ee801dccf24)

note these are the oauth services I have setup

**Reply by mikekuzak** [reply]
That's would be a good start. Otherwise it might be confusing for some ppl. I don't think currently you can hide the email section. Can you?

**Reply by danny-avila** [reply]
> I don't think currently you can hide the email section.

Not currently but this is an easy enhancement. Either me or one of the active contributors will pick it up!

**Reply by danny-avila** [reply]
I've opened an issue for this!

**Reply by danny-avila** [reply, marked as answer]
This is merged now btw
